### ISSUE TITLE
why is the file extension "Love"

### ISSUE THREAD

**Issue by jsienkiewicz13** [KyYay/CelLua-Machine#4]

**Comment by ismael1222** [comment]
cuz you open it with LOVE, i mean, literally.
To open [install the game engine here](https://love2d.org/)

**Comment by TheoCGaming** [comment]
Correct, LOVE is the engine the game runs on, and it requires the use of said engine to run the game.

**Comment by TheoCGaming** [comment]
@KyYay issue needs to be closed
